### ISSUE TITLE
Does this repo contain your training code

### ISSUE THREAD

**Issue by arifsaeed** [castorini/rank_llm#135]
I wanted to see how you implemented the loss function and back-propogated against it, but i cant find it in the code.  Is the training code available to look at

**Comment by ronakice** [comment]
This repository currently only contains inference code, focusing on a wide-suite of tasks. But we plan to make some training stuff available soon, stay tuned!
